src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else {
+    return days * 40;
+  }
```

Comment:
Hi, good job! but there is a checklist that we need to stick to. Please, check it.
Points 1 and 3.